posta/models.py

```diff
@@ -54,6 +50,9 @@ class Meta:
     # Flag per i messaggi cancellati (perche' obsoleti)
     eliminato = models.BooleanField(default=False, null=False)
 
+    priorita = models.IntegerField(default=0)
```

Comment:
A cosa serve `priorita`?

Reply:
quando un invio fallisce viene incrementato, la query che processa la coda ha un order by priorita, i messaggi con la priorità più bassa vengono selezionati prima

Reply:
OK. Possiamo implementare un conto alla rovescia (`tentativi_rimasti`) oppure rinominare il campo in modo che sia piu' chiaro (`tentativi_effettuati`)? Nel secondo caso e' importante che prevediamo un numero massimo di tentativi: se l'invio fallisce 5 volte, e' probabile che non riusciremo mai a inviare quel messaggio.

Reply:
In merito a questa situazione, in caso di messaggio non inviato dopo 5 tentativi, sarebbe necessario che giunga un feedback in merito a questo fatto all'utente. Penso tuttavia che gli scenari possibili siano effettivamente molti (utente che riceve un messaggio automatico dal sistema, utente che riceve un messaggio in seguito ad azione di un altro utente, utente che manda messaggi massivi,...). Lo si lascia morire e basta?

Reply:
@luca-dex a mio avviso e' una complicazione non necessaria, per ora. Pensa agli invii di massa: non vogliamo mica che il presidente nazionale riceva mille notifiche perche' mille volontari da qualche parte in Italia hanno un indirizzo email errato. Piuttosto, e' gia' un requisito che abbiamo discusso il fatto che il mittente possa verificare lo stato dello smistamento dai dettagli del messaggio nella sezione posta inviata, similmente a quanto e' gia' possibile fare ora. Che ne pensi?

Reply:
Sì, tuttavia, al momento, per gli invii di massa non viene mostrato lo stato di tutti i messaggi ma solo di un sottoinsieme dei destinatari a cui è stato inviato il messaggio. Comunque ok per lasciarlo ad una futura valutazione

Reply:
@luca-dex quello che scrivi sembra davvero materiale per un bellissimo nuovo task JIRA :)

Reply:
`priorita` sarebbe al momento "tentativi falliti", e nella query di spool si può mettere un limite, tuttavia se di default mettiamo 1 invece di zero è possibile usare 0 per mail con priorità di invio più alta

Reply:
priorita sarebbe al momento "tentativi falliti", e nella query di spool si può mettere un limite, tuttavia se di default mettiamo 1 invece di zero è possibile usare 0 per mail con priorità di invio più alta